wapitiCore/net/scope.py

```diff
@@ -46,29 +46,37 @@ def name(self) -> str:
         return self._scope
 
     def check(self, resource: Union[Request, str]) -> bool:
+
+        checked = None
```

Comment:
I don't see any benefit of having that variable instead of using return statements. If the only purpose is to avoid the linter it seems ok to me to use a `# pylint: disable=R0911` because refactoring doesn't seem coherent here.

If you prefer to keep the variable, protect each branch with an `elif` instead as only one scope can be chosen

Reply:
Yes it is to avoid linter